profile-widget/profilewidget2.cpp

```diff
@@ -1276,6 +1276,7 @@ void ProfileWidget2::setAddState()
 	DivePlannerPointsModel *plannerModel = DivePlannerPointsModel::instance();
 	connect(plannerModel, &DivePlannerPointsModel::dataChanged, this, &ProfileWidget2::replot);
 	connect(plannerModel, &DivePlannerPointsModel::cylinderModelEdited, this, &ProfileWidget2::replot);
+	connect(plannerModel, &DivePlannerPointsModel::modelReset, this, &ProfileWidget2::pointsReset);
```

Comment:
comment on the commit message - there's one typo "The assumed single..." -- but overall I struggle a bit with understanding what it says. Would you mind rephrasing a bit?